ksql-clickstream-demo/README.md

```diff
@@ -15,11 +12,4 @@ This demo focuses on building real-time analytics of users to determine:
 
 The demo uses standard streaming functions (i.e., min, max, etc), as well as enrichment using child tables, table-stream joins and different types of windowing functionality.
```

Comment:
should be 'stream-table join'

Reply:
all `.md` has been removed. changed in https://github.com/confluentinc/ksql/pull/832/files/9ad4adffabb7ac39ef0638b9ce292cca912baf4a#diff-21c12b62d71bcfca8656a765b65a581f